src/Aquifer.Public.API/Endpoints/ParentResources/List/Endpoint.cs

```diff
@@ -0,0 +1,89 @@
+using Aquifer.Common.Services.Caching;
+using Aquifer.Data;
+using Aquifer.Public.API.Helpers;
+using FastEndpoints;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.ParentResources.List;
+
+public sealed class Endpoint(AquiferDbContext _dbContext, ICachingLanguageService _cachingLanguageService)
+    : Endpoint<Request, List<Response>>
+{
+    public override void Configure()
+    {
+        Get("/resources/parent-resources");
+        Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.TenMinutesInSeconds));
+        Summary(s =>
+        {
+            s.Summary = "Gets a list of parent resources for a given language.";
+            s.Description =
+                "Returns all parent resources for a given language including the count of resources associated with each parent.";
+        });
+    }
+
+    public override async Task HandleAsync(Request req, CancellationToken ct)
+    {
+        var languageId = await GetLanguageIdAsync(req.LanguageId, req.LanguageCode, ct);
+
+        var query = $"""
```

Comment:
This is mostly a copy/paste of the Internal API's code but I removed the default fallback to English for images and videos as well as a few columns we didn't need.